repair/row_level.cc

```diff
@@ -1639,6 +1639,49 @@ class repair_meta {
     }
 };
 
+static future<stop_iteration> repair_get_row_diff_with_rpc_stream_process_op(
+        gms::inet_address from,
```

Comment:
This won't compile because of the static (unused function).

Reply:
Other than squashing with the user, any other suggestions? I do want to put them in different commits.